prometheus/exporter/job_exporter.py

```diff
@@ -83,11 +88,12 @@ def genJobMetrics(logDir, gpuMetrics):
         outputFile.write(containerMemPerc)
 
 def main(argv):
+    logDir = argv[0]
+    timeSleep = int(argv[1])
     while(True):
-        logDir = argv[0]
-        timeSleep = int(argv[1])
+        print("job exporter running")
```

Comment:
>print("job exporter running") [](start = 8, length = 29)

move to ln 93, before the while loop? are you going to print it in every iteration?

Reply:
yes hope to print evey iteration, will move

Reply:
if so, suggest to add a counter or timestamp

---
In reply to: [181279874](https://github.com/Microsoft/pai/pull/472#discussion_r181279874) [](ancestors = 181279874)